frontend/proof-of-existence/src/views/certify/SelectWalletModel.vue

```diff
@@ -0,0 +1,162 @@
+<script setup lang="ts">
+import { onMounted, ref, toRefs } from "vue";
+import { empowerchain } from "@empowerplastic/empowerchain-ts-client";
+import { Wallet } from "@/types/enums";
+import { HttpEndpoint } from "@cosmjs/tendermint-rpc";
+
+//Modal props
+defineProps({
+  showModal: Boolean,
+});
```

Comment:
Heads up!  A tip is to use _type declaration_ when using `defineProps`, then we can be consistent using typescript to type things. More info: https://vuejs.org/api/sfc-script-setup.html#type-only-props-emit-declarations



```suggestion
export interface IProps {
  showModal: boolean
}

defineProps<IProps>()
```